content.md

```diff
@@ -219,18 +207,12 @@ Some library cards have multiple Discipline symbols on the attribute bar
 ![](bin/media/image11.png)
 
 
-
-> **DECK CONSTRUCTION**\
-> **Group:** Each vampire belongs to a specific group, identified by a
-> number above the upper left corner of the text box. A Methuselah's
-> crypt must be built using vampires from a single group or from two
-> consecutive groups. This does not restrict a Methuselah from stealing
-> vampires from other groups through play, however.
-
-
 ![](bin/media/image12.png)
 
-> **ADVANCED RULES**\
+> **Group:** Each vampire belongs to a specific group, identified by a
+> number above the upper left corner of the text box.
```

Comment:
It should at least link to "Deck Construction", otherwise for a brand new player it's a confusing piece of information ("right, group, but what is this for?")

Reply:
agreed :) I don't know how to make links yet, as I am very new to GitHub. How can I do that good?